test/fidelity/damaged-helmet-hdr-pmrem/index.html

```diff
@@ -0,0 +1,37 @@
+<!--
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<!doctype html>
+<html>
+  <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <title>&lt;model-viewer&gt; Fidelity Test - Damanged Helmet HDR PMREM</title>
+
+  <style>
+    body { margin: 0; }
+  </style>
+</head>
+<body>
+  <model-viewer src="../../../examples/assets/DamagedHelmet/DamagedHelmet.gltf"
+      background-image="../../../examples/assets/aircraft_workshop_01_4k.hdr"
+      experimental-pmrem
+      style="width: 768px; height: 768px;">
+  </model-viewer>
+  <p>This model demonstrates the alpha blending behavior of a semi-transparent "container" object surrounding several opaque objects.</p>
```

Comment:
Nit: this refers to the other opacity test

Reply:
👍 will fix